backend/controllers/org.controller.ts

```diff
@@ -10,20 +41,47 @@ export const organizationTest = async (req: Request, res: Response) => {
 
 // Create a new Organization
 export const createNewOrganization = async (
-  req: Request,
+  // req: Request,
+  req: RequestExtended,
   res: Response,
   next: NextFunction,
 ) => {
   try {
-    // const userId = req.userId; // get userId from the request
-    const userId = "3"; // NOTE: Placeholder, remove later
     // Validate the organization data
     const validatedOrganization = OrganizationCreateSchema.parse(req.body);
 
+    const loggedInUserId = req.userID;
+    // If we want to make image upload mandatory upon organization creation
+    // if (!req.file) {
+    //   throw new AppError(
```

Comment:
I think let's leave it optional for now, but in the future, this may change because we may choose to add 'organization' profile pages that people can visit.